Rules/CTF/Scripts/CTF.as

```diff
@@ -685,6 +685,11 @@ void Reset(CRules@ this)
 	this.set("core", @core);
 	this.set("start_gametime", getGameTime() + core.warmUpTime);
 	this.set_u32("game_end_time", getGameTime() + core.gameDuration); //for TimeToEnd.as
+
+	for (int team = 0; team < this.getTeamsCount(); team++)
```

Comment:
What about other gamemodes? Why is this only in CTF.as?

Reply:
Because quarries dont exist in TTH, and in sandbox quarry output doesn't matter. I'll make it decrease output only in CTF modes.

Reply:
But what if this changes? You should implement this in a way that works for any gamemode that needs this behaviour.